website/static/css/pages/remove-contributor-page.css

```diff
@@ -0,0 +1,4 @@
+.col-md-4 {
+    padding-top: 20px;
+    padding-right: 15px;
+}
```

Comment:
> Interesting! Looks like it's part of webpack in our case?

Correct. webpack has an implementation of the CommonJS standard (similar to how RequireJS is an implementation of AMD).